python/swiftnav/nav_msg.pyx

```diff
@@ -40,3 +43,57 @@ cdef class NavMsg:
   def process_subframe(self, e):
```

Comment:
Please, take into account that In order to implement L2C capability monitor (please, see PR https://github.com/swift-nav/libswiftnav/pull/294/ still not merged) I updated process_subframe. Now it takes pointer to gps_l1ca_decoded_data_t as the 2nd parameter. So I guess tmp type should be changed. However, I believe this should be done after my PR be merged.